cdap-api-common/src/main/java/io/cdap/cdap/api/common/ConstantsUtil.java

```diff
@@ -24,8 +24,14 @@
 /**
  * Constants used by different systems are all defined here.
  */
-public class Constants {
-  public static final List<Integer> RETRYABLE_HTTP_CODES = Collections.unmodifiableList(
-    Arrays.asList(HttpURLConnection.HTTP_BAD_GATEWAY, HttpURLConnection.HTTP_UNAVAILABLE,
-                  HttpURLConnection.HTTP_GATEWAY_TIMEOUT));
+public class ConstantsUtil {
```

Comment:
This is not a good name. Is this class supposed to have `ApiConstants`? Or `HttpConstants`? Calling it util doesn't make it any more clear.

Reply:
Also, this is an API class, we shouldn't just rename it, as it breaks API compatibility. If we want to avoid name collision, we should rename the internal one, not the API one.

Reply:
This class was added recently in https://github.com/cdapio/cdap/pull/14777, does not exist in any CDAP release yet.
Do we care about API compatibility if it only exists in develop right now? Is it ok to rename to `HttpConstants`?

Reply:
If it's not part of 6.8, it should be fine I believe

Reply:
But I see it in 6.8

Reply:
https://github.com/cdapio/cdap/blob/release/6.8/cdap-api-common/src/main/java/io/cdap/cdap/api/common/Constants.java

Reply:
But not in 6.8.0: https://github.com/cdapio/cdap/blob/v6.8.0/cdap-api-common/src/main/java/io/cdap/cdap/api/common/Constants.java - not found

Reply:
right, it's not a released version. will go into 6.8.1, but planning to cherrypick my fix into release/6.8 as well

Reply:
fixed